### ISSUE TITLE
bosh search returns nothing

### ISSUE THREAD

**Issue by yarikoptic** [boutiques/boutiques#658]
## Expected Behavior

find (matching) boutiques

## Current Behavior

finds nothing:

```shell
❯ bosh search -v
[ INFO ] Using Query Line:  AND (/.*%5BBb%5D%5BOo%5D%5BUu%5D%5BTt%5D%5BIi%5D%5BQq%5D%5BUu%5D%5BEe%5D%5BSs%5D.*/)
[ INFO ] Using Zenodo endpoint https://zenodo.org
[ INFO (200) ] Search successful for query "boutiques"
[ INFO ] Showing 0 of 0 result(s).
```

**Comment by glatard** [comment]
Thanks for reporting @yarikoptic, this is fixed in #659 and will be available in next release.

**Comment by glatard** [comment]
This is now available in release 0.5.26.
